playwright/tests/index.ts

```diff
@@ -64,10 +63,10 @@ class Server implements Disposable {
     while (this.portNumber === null) {
       await sleep(10)
     }
-    return `http://${this.host}:${this.portNumber}`
+    return `http://${this.host}:${this.portNumber.toString()}`
   }
 
-  public async [Symbol.dispose] () {
+  public [Symbol.dispose] (): Promise<void> {
```

Comment:
This is not async and therefore does not return a promise. Why does ESLint not catch this?